README.md

```diff
@@ -162,6 +162,43 @@ A `HeaderModel_` class is generated that subclasses HeaderModel and implements t
 
 [More Details](https://github.com/airbnb/epoxy/wiki/ViewHolder-Models)
 
+#### From ViewBinding
+
+If you use ViewBinding in your project, you have two ways to create a model that can use your \*Binding class:
+1. Using annotation processing (like examples above)
+```kotlin
+@EpoxyModelClass(layout = R.layout.view_binding_holder_item)
+abstract class ItemViewBindingEpoxyHolder : ViewBindingEpoxyModelWithHolder<ViewBindingHolderItemBinding>() {
+
+    @EpoxyAttribute lateinit var listener: () -> Unit
+    @EpoxyAttribute lateinit var title: String
+
+    override fun ViewBindingHolderItemBinding.bind() {
+        title.text = this@ItemViewBindingEpoxyHolder.title
+        title.setOnClickListener { listener() }
+    }
+}
+``` 
+where [ViewBindingEpoxyModelWithHolder](https://github.com/airbnb/epoxy/blob/master/kotlinsample/src/main/java/com/airbnb/epoxy/kotlinsample/helpers/ViewBindingEpoxyModelWithHolder.kt) - base class special for this case 
+
+2. Using data class
+```kotlin
+// This does not require annotations or annotation processing.
+// The data class is required to generated equals/hashcode which Epoxy needs for diffing.
+// Views are easily declared via property delegates
+data class ItemViewBindingDataClass(
+    val title: String
+) : ViewBindingKotlinModel<DataClassViewBindingItemBinding>(R.layout.data_class_view_binding_item) {
+
+    override fun DataClassViewBindingItemBinding.bind() {
+        title.text = this@ItemViewBindingDataClass.title
+    }
+    
+}
+```
+where [ViewBindingKotlinModel](https://github.com/airbnb/epoxy/blob/master/kotlinsample/src/main/java/com/airbnb/epoxy/kotlinsample/helpers/ViewBindingKotlinModel.kt) - base class special for this case
```

Comment:
In ViewBindingKotlinModel there is `R.id.epoxy_viewbinding` that can be unknown for those who is going to copy-paste this base class.
Maybe I should add some info about it? 
Like "for correct usage of this base class, add <item name="epoxy_viewbinding" type="id" /> in res/xml/ids.xml in your project"